test/python/apps/exchange.py

```diff
@@ -44,6 +44,23 @@ class Rate(IntEnum):
     FLOATING = 0x01
 
 
+class Signature_Computation(Enum):
```

Comment:
Please stick to[ Pep8 naming recommendation](https://peps.python.org/pep-0008/#class-names).